GTBitsOfGood/national-npp
# Added national admin view

## What is this?

Added national admin view of chapters and nonprofits, as well as button to delete them.


## Setup



## Steps to Test


Go to /natlAdmin and toggle the tab between nonprofits and chapters. Triple dots are the delete function which pops up with a confirm alert. Pressing delete closes the alert for now.

## Associated Issues

Closes #79


## Additional Instructions
